models/pagination_and_sorting.go

```diff
@@ -1,58 +1,115 @@
 package models
 
-import "fmt"
+import (
+	"github.com/cockroachdb/errors"
+)
+
+const COUNT_ROWS_LIMIT = 1000
 
 type PaginationAndSorting struct {
 	OffsetId string
 	Sorting  SortingField
 	Order    SortingOrder
 	Limit    int
-	Previous bool
-	Next     bool
 }
 
 func NewDefaultPaginationAndSorting(sortColumnName string) PaginationAndSorting {
 	return PaginationAndSorting{
-		Sorting: SortingField(sortColumnName),
+		Sorting: SortingFieldFrom(sortColumnName),
 		Order:   SortingOrderDesc,
 		Limit:   100,
 	}
 }
 
-type (
-	SortingField string
-	SortingOrder string
+type SortingField int
+
+const (
+	SortingFieldUnknown SortingField = iota
+	SortingFieldCreatedAt
+	SortingFieldUpdatedAt
 )
 
+func (sf SortingField) String() string {
+	switch sf {
+	case SortingFieldCreatedAt:
+		return "created_at"
+	case SortingFieldUpdatedAt:
+		return "updated_at"
+	default:
+		return "unknown"
+	}
+}
+
+func SortingFieldFrom(s string) SortingField {
```

Comment:
Disclaimer: a limitation is that this implementation only works as-is if the field on which we order in the DB has the same name as the field on the DTO. This is fine for now I think, and we'd just need to introduce a mapping table in the future if we need something more advanced.